code/modules/antagonists/arcfiend/arcfiend.dm

```diff
@@ -47,6 +56,12 @@
 		var/points = min((MAX_ARCFIEND_POINTS - src.points), add_points)
 		. = ..(points, target_ah_type)
 
+	proc/get_arcfiend_power(var/total_power = 0)
+		if (total_power)
+			return src.lifetime_energy
+		else
+			return src.points
+
```

Comment:
should be two procs, but at that point it also has no point vs just checking vars
```suggestion
```